### DISCUSSION TITLE
Provide lists dropdown inside Star button [Android]

### DISCUSSION THREAD

**Discussion by ghost** [community/community#36616] [Other Feature Feedback, Questions, & Ideas]
Under Android useragent, star button inside a repository does not have "List" dropdown. But on desktop mode, it does.

Android :
![Screenshot_20221019](https://user-images.githubusercontent.com/83843264/196598567-273695bf-7484-4ff9-9a78-08fc71322eec.png)

Desktop mode:
![Screenshot_20221019](https://user-images.githubusercontent.com/83843264/196598884-30d4969d-8156-4ffb-a927-a512597218ce.png)

**Comment by Suesexy453** [comment]
Finally found my fikes

**Reply by Suesexy453** [reply]
Html.www.suestrades.com
